File: Java Programs/Practice/FibonacciNumberGenerate.java
import java.util.*;

public class FibonacciNumberGenerate
{
	public static void main(String args[])
	{
		Scanner sc = new Scanner(System.in);
		
		System.out.print("\n Enter the number of terms : ");
		int n = sc.nextInt();
		
		System.out.print("\n The fibonacci series is : ");
		
		int f1 = 0, f2 = 1;
		
		for(int i = 0; i < n; i++)
		{
			System.out.print(f1 + " ");
			
			int fn = f1 + f2;
			f1 = f2;
			f2 = fn;
		}
		
		System.out.println();
		
		sc.close();
	}
}

File: Python Programs/fibonacci_number_generate.py
n = int(input("Enter the no. of terms : "))

print("The fibonacci numbers are :", end = " ")

f1 = 0
f2 = 1

for i in range(1, n + 1):
    print(f1, end = " ")
    fn = f1 + f2
    f1 = f2
    f2 = fn
